addons/beehave/nodes/beehave_root.gd

```diff
@@ -69,16 +73,3 @@ func get_last_condition_status():
 			return "RUNNING"
 	return ""
 
-func enable():
```

Comment:
We should not remove these methods. We still want to be able to manually enable or disable the behaviour tree programmatically.  Unless there is a different way to do this now?